test/test_linalg.py

```diff
@@ -674,6 +675,122 @@ def test_norm_fastpaths(self, device):
         expected = torch.pow(x.pow(3).abs().sum(1), 1.0 / 3.0)
         self.assertEqual(result, expected)
 
+    @skipCPUIfNoLapack
+    @skipCUDAIfNoMagma
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
The test suite now requires NumPy so we no longer need checks like this

Reply:
e71ac6c6cc